petabvis/main.py

```diff
@@ -105,26 +106,30 @@ def add_plots(self):
         Returns:
             List of PlotItem
         """
-        self.clear_QSplitter()
-        self.visu_spec_plots.clear()
+        self.clear_qsplitter()
+        self.vis_spec_plots.clear()
 
         if self.visualization_df is not None:
             # to keep the order of plots consistent with names from the plot selection
-            indexes = np.unique(self.visualization_df[ptc.PLOT_ID], return_index=True)[1]
-            plot_ids = [self.visualization_df[ptc.PLOT_ID][index] for index in sorted(indexes)]
+            indexes = \
+            np.unique(self.visualization_df[ptc.PLOT_ID], return_index=True)[1]
+            plot_ids = [self.visualization_df[ptc.PLOT_ID][index] for index in
+                        sorted(indexes)]
```

Comment:
Yes, actually i already replaced all instances where i used the workaround with indexes to `list(...unique())` but i forgot to add them to the pull request. I already updated it in the replicates branch. There, I also updated the warnings function.
How is this usually handled, since both branches changed the code?

Reply:
Then there will be merge conflicts that will need to be resolved. Please resolve all suggestions here that you've already applied, as there is no need to do the change here if you've done it in your other branch. Once this PR is merged, please pull and merge into your branch, then resolve any conflicts.

Reply:
> then resolve any conflicts

That might be a bit painful now. I assumed everything was merged. :grimacing: 

Depending on how much you changed meanwhile, it may be easier to merge this one first, and then apply the changes again.

Reply:
Thankfully, I didn't add too much since the pull request. I guess pulling after the request merged and reapplying my changes shouldn't take too long I hope.